include/dpp/application.h

```diff
@@ -209,6 +212,13 @@ class DPP_EXPORT app_team {
 	snowflake owner_user_id;
 };
 
+/**
+ *  * @brief Configuration object for an app installation
```

Comment:
What happened here with the comments lol

Reply:
oof lol